src/app/model/vehicle.ts

```diff
@@ -0,0 +1,9 @@
+export interface Vehicle {
```

Comment:
Can an interface specify fields? If not this might be a compelling reason to use getters instead